src/components/scenes/SweepPrivateKeyCalculateFeeScene.tsx

```diff
@@ -200,6 +200,8 @@ const SweepPrivateKeyCalculateFeeComponent = (props: Props) => {
     'SweepPrivateKeyCalculateFeeComponent'
   )
 
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const extraData = React.useMemo(() => [], [transactionState, exchangeRates])
```

Comment:
Why not just return `({ transactionState, exchangeRates})` in the memo function? I get that you're just trying to get the FlatList to re-render, but why not propagate the data just to avoid more questions when someone reads this code later?